### ISSUE TITLE
error: unknown shared.MongoSessionOpts field

### ISSUE THREAD

**Issue by trollknurr** [dcu/mongodb_exporter#37]
Hello!

Trying to compile latest master, got this error:
```
collector/mongodb_collector.go:30: unknown shared.MongoSessionOpts field 'TlsCertificateFile' in struct literal
collector/mongodb_collector.go:31: unknown shared.MongoSessionOpts field 'TlsPrivateKeyFile' in struct literal
collector/mongodb_collector.go:32: unknown shared.MongoSessionOpts field 'TlsCaFile' in struct literal
collector/mongodb_collector.go:33: unknown shared.MongoSessionOpts field 'TlsHostnameValidation' in struct literal
```

Compile using this container [docker container](https://hub.docker.com/_/golang/) tag 1.6

P.S. install missing dependence with command

`go get -u github.com/golang/glog`

**Comment by thematt** [comment]
i have same error

**Comment by martinhoefling** [comment]
https://github.com/dcu/mongodb_exporter/pull/38 should fix this.

**Comment by deepthawtz** [comment]
Fixed by #36
